### PR TITLE
Use download link requests

### PR DESCRIPTION
**Pull request by MGibson1**
# Overview

Implements download link request scheme from bitwarden/jslib#288 and bitwarden/server#1174.

# Files Changed
* **access.component.ts**: request download link.
* **messages.json**: report if file cannot be found due to missing download link.

# Draft reason
Depends on bitwarden/jslib#288

### CODE REVIEW

**Review by cscharf — COMMENTED** [review]
Should be good to go once jslib rolls in.

**Comment by MGibson1** [comment]
holding off on this PR for bitwarden/jslib#289

**Review by cscharf — DISMISSED** [review]

**Review by cscharf — APPROVED** [review]

**Pull request merged**
